velox/functions/prestosql/Reduce.cpp

```diff
@@ -168,6 +172,9 @@ class ReduceFunction : public exec::VectorFunction {
           nullptr,
           &localResult);
     }
+    if (flatArray->rawNulls()) {
+      localResult->addNulls(flatArray->rawNulls(), rows);
```

Comment:
@yma11 I'm seeing errors that seem to be coming from this change:

```
VeloxRuntimeError:  addNulls not supported presto.default.reduce
	at Unknown.# 0  facebook::velox::VeloxException::VeloxException(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, facebook::velox::VeloxException::Type, std::basic_string_view<char, std::char_traits<char> >)(Unknown Source)
	at Unknown.# 1  void facebook::velox::detail::veloxCheckFail<facebook::velox::VeloxRuntimeError, char const*>(facebook::velox::detail::VeloxCheckFailArgs const&, char const*)(Unknown Source)
	at Unknown.# 2  facebook::velox::ConstantVector<facebook::velox::ComplexType>::addNulls(unsigned long const*, facebook::velox::SelectivityVector const&)(Unknown Source)
	at Unknown.# 3  facebook::velox::functions::(anonymous namespace)::ReduceFunction::apply(facebook::velox::SelectivityVector const&, std::vector<std::shared_ptr<facebook::velox::BaseVector>, std::allocator<std::shared_ptr<facebook::velox::BaseVector> > >&, std::shared_ptr<facebook::velox::Type const> const&, facebook::velox::exec::EvalCtx&, std::shared_ptr<facebook::velox::BaseVector>&) const(Unknown Source)
```

Reply:
Found that this is already resolved by [6886](https://github.com/facebookincubator/velox/pull/6886). Thanks @bikramSingh91 for the fix.